docs/components/connectors/out-of-the-box-connectors/embeddings-vector-db.md

```diff
@@ -0,0 +1,147 @@
+---
+id: embeddings-vector-db
+title: Vector Database Connector
+sidebar_label: Vector Database
+description: Embed, store and retrieve LLM embeddings and store them into vector databases.
+---
+
+The **Vector Database Connector** allows embedding, storing and retrieving Large Language Model (LLM) embeddings. This enables building
+AI-based solutions for your organizations, such as context document search, long-term LLM memory and/or agentic AI interaction.
+
+:::note
+The **Vector Database Connector** uses the [Langchain4J](https://docs.langchain4j.dev/) library. Data models and possible implementations are limited
+to the latest stable released Langchain4j library.
+:::
+
+## Prerequisites
+
+Before using the **Vector Database Connector**, it is highly recommended to understand the
+[concept of LLM embeddings](https://huggingface.co/spaces/hesamation/primer-llm-embedding).
+
+To start using the **Vector Database Connector**, you need to have access to supported LLM embeddings API to convert
+documents content into vectorized embeddings form. You will also need to have write access to supported databases.
+
+## Create a Connector task
+
+import ConnectorTask from '../../../components/react-components/connector-task.md'
+
+<ConnectorTask/>
+
+## Operations
+
+### Embed document
+
+The **Embed document** operation performs the following steps: [consumes a document](#embedding-document-configuration),
+parses it depending on a file format, optionally [splits into text chunks](#splitting), converts chunks into a vector form
+with an LLM help, and finally, stores produced vectors into a vector database.
+
+To perform this operation, enter the following:
+
+- **Operation** dropdown: **Embed document**.
+- **Embedding model** section: refer to the [relevant section](#embedding-models).
+- **Vector store** section: refer to the [relevant section](#vector-stores).
+- **Document** section: refer to the [relevant section](#embedding-document-configuration).
+
+As a result of this operation you will get an array of created embeddings,
```

Comment:
Maybe we can elaborate what is in this array - are these the chunk IDs?

Reply:
Good point. How about:

```suggestion
As a result of this operation you will get an array of created embeddings IDs,
```

?